network-api/networkapi/wagtailpages/templates/wagtailpages/homepage.html

```diff
@@ -11,18 +11,15 @@
 
 {% block content %}
 <div class="container">
-  <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center homepage-feature">
-    <div class="col-md-6 mb-3 d-sm-none">
-      {% image page.hero_image fill-1152x770-c100 format-jpeg class="full-bleed-xs" %}
+  <div class="row mb-5 mt-0 mt-sm-5 d-flex align-items-center flex-sm-row-reverse homepage-feature">
+    <div class="col-sm-6 mb-3 mb-sm-0">
```

Comment:
This line change seems to cause the Percy regression. Is that an intentional change?

Reply:
Yes, these changes were intentional.